app/backend/src/clj/teet/cooperation/cooperation_commands.clj

```diff
@@ -72,6 +74,21 @@
                                     [:cooperation.application/response :db/id])]
     (= response-id current-response-id)))
 
+(defn application-response-notification-tx
+  "Add notification about new Application response,
+  return a notification transaction map or empty map if receiver or sender is empty"
+  [db user activity-user project application-id]
+  (if
+    (and
+      (some? user)
+      (some? activity-user))
+    (notification-db/notification-tx
+      db {:from user
+          :to activity-user
+          :target application-id
+          :type :notification.type/cooperation-response-to-application-added
+          :project project}) {}))
```

Comment:
Add newline before the empty `{}`. It wasn't immediately clear that this was the else expression for the `if`.